app/helpers/collection_time_helper.rb

```diff
@@ -44,4 +44,16 @@ def previous_collection_start_year
   def previous_collection_start_date
     current_collection_start_date - 1.year
   end
+
+  def currently_crossover_period?
+    # generally the crossover period ends on the first Friday in June, but there may be arbitrary exceptions such as in 2023
+    today = Time.zone.today
+    if today.year == 2023
+      today.between?(Time.zone.local(2023, 4, 1).to_date, Time.zone.local(2023, 6, 9).to_date)
+    else
+      crossover_end_date = Time.zone.local(today.year, 6, 1).to_date
+      crossover_end_date += 1 until crossover_end_date.friday?
+      today.between?(Time.zone.local(today.year, 4, 1).to_date, crossover_end_date)
+    end
+  end
```

Comment:
Is this the same as `FormHandler#in_crossover_period?`?

Reply:
aha. hadn't seen that.
very good.
Can I get a sense check though as this might be important for the future depending on what we're using this method for (the FormHandler one). I would expect that that method would return false for all times on the final day of the crossover period.
i.e. Time.zone.now returns the time now, but the end_date for the forms appears to be 00:00 on that day, so now < end_date will be false.
Shall I make a tech debt/bug ticket to change to now.to_date <= end_date.to_date?
I am fairly confident in that change, but I may not want to slow down a 23/24 ticket at this time writing tests.